plantcv/plantcv/print_results.py

```diff
@@ -1,18 +1,169 @@
 # Print Numerical Data
 
+from plantcv.plantcv import outputs
 
-def print_results(filename, header, data):
+
+def print_results(filename):
     """Print result table
 
     Inputs:
     filename = filename
-    header   = result data table headers
-    data     = result data table values
 
     :param filename: str
-    :param header: list
-    :param data: list
     :return:
     """
-    print('\t'.join(map(str, header)))
-    print('\t'.join(map(str, data)))
+    result = open(filename, "a")
+    if 'bound_horizontal' in outputs.measurements:
+        bound_header = [
+            'HEADER_BOUNDARY_H',
+            'height_above_bound',
+            'height_below_bound',
+            'above_bound_area',
+            'percent_above_bound_area',
+            'below_bound_area',
+            'percent_below_bound_area']
+        bound_horizontal_dict = outputs.measurements['bound_horizontal']
+
+        result.write('\t'.join(map(str, bound_header)) + '\n')
+        result.write('BOUNDARY_H_DATA')
+
+        for k, v in bound_horizontal_dict.items():
+            result.write('\t' + str(v))
+        result.write('\n')
+
+    if 'bound_vertical' in outputs.measurements:
+        bound_header = [
+            'HEADER_BOUNDARY_V',
```

Comment:
The measurements here are not programmed into the `plantcv-pipeline.py` script, but I think we need to fix that over there as opposed to here.